sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.autoconfigure;
+
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter;
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporterBuilder;
+import io.opentelemetry.exporter.prometheus.PrometheusCollector;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.export.IntervalMetricReader;
+import io.prometheus.client.exporter.HTTPServer;
+import java.io.IOException;
+import java.util.Collections;
+
+final class MetricExporterConfiguration {
+
+  static boolean configureExporter(
+      String name,
+      ConfigProperties config,
+      boolean metricsAlreadyRegistered,
+      SdkMeterProvider meterProvider) {
+    switch (name) {
+      case "otlp":
+      case "otlp_metrics":
+        if (metricsAlreadyRegistered) {
```

Comment:
it's a little annoying that this same check has to be done twice, but I don't see any easy away around it without some sort of ugly nesting of switch statements